core/modules/qproc/QuerySession.h

```diff
@@ -244,19 +245,18 @@ class QuerySession {
     */
     query::SelectStmtPtr _stmtMerge;
 
-    bool _hasMerge{false};
-    bool _isDummy{false}; ///< Use dummy chunk, disabling subchunks or any real chunks
+    bool _hasMerge = false;
+    bool _isDummy = false; ///< Use dummy chunk, disabling subchunks or any real chunks
     std::string _tmpTable;
     std::string _resultTable;
     std::string _error;
-    int _isFinal{0}; ///< Has query analysis/optimization completed?
+    int _isFinal =0; ///< Has query analysis/optimization completed?
 
     ChunkSpecVector _chunks; ///< Chunk coverage
     std::shared_ptr<QueryPluginPtrVector> _plugins; ///< Analysis plugin chain
 
-    /// Maximum number of chunks in an interactive query. TODO: DM-10273 put in config file.
-    int const _interactiveChunkLimit{10};
-    bool _scanInteractive{true}; ///< True if the query can be considered interactive.
+    int const _interactiveChunkLimit = 10; ///< Maximum number of chunks in an interactive query.
```

Comment:
The same default is set in `CzarConfig`. Do you think it would worth an effort to define it in one location? If not then it's probably okay.

Reply:
Unit tests use it. i added a comment to that affect.